prdoc/pr_2903.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: "implement ConversionToAssetBalance in asset-rate"
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Implements the `ConversionToAssetBalance` trait to the asset-rate pallet.
+
+      Previously only the `ConversionFromAssetBalance` trait was implemented, which would allow to convert an asset balance into the corresponding native balance
```

Comment:
```suggestion
      Previously only the `ConversionFromAssetBalance` trait was implemented, which would allow to convert an asset balance into the corresponding native balance.
```